configs/AWS-Hpc7a-EPYC4-EFA/system_definition.yaml

```
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: AWS-Hpc7a-EPYC4-EFA
+  manufacturer:
+    vendor: AWS
+    name: ParallelCluster3.7.2-Hpc7a
+  processor:
+    vendor: AMD
+    name: EPYC-Zen4
+    ISA: x86_64
```

I don't think this is specific enough for ISA -- `zen3` and `skylake` do not share an instruction set, even though both are derived from `x86_64`.

What about 

```suggestion
    ISA_family: x86_64
    ISA: zen4
```

---

resolved.